lib/Rex/CLI.pm

```diff
@@ -455,6 +454,30 @@ sub __help__ {
 
 }
 
+# Check whether Rex/Template.pm
+# or Rex/Template/NG.pm modules are loaded. If there are,
+# they should be present in the %INC hash.
+sub check_template_modules {
```

Comment:
We have `Rex::deprecated` to mark things deprecated in Rex core in a uniform way. That might come in handy here too. Or perhaps during `import` time of Rex::Template and Rex::Template::NG.